cmd/experimental/kueue-viz/README.md

```diff
@@ -0,0 +1,83 @@
+# Build and Run locally
+
+## Prerequisites
+You need a kubernetes cluster running kueue.
+If you don't have a running cluster, you can create one using kind and install kueue using helm.
+
+```
+kind create cluster
+kind get kubeconfig > kubeconfig
+export KUBECONFIG=$PWD/kubeconfig
+helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue \
+            --version="v0.9.1" --create-namespace --namespace=kueue-system
+```
+
+## Build
+Clone the kueue repository and build the projects:
+
+```
+git clone https://github.com/kubernetes-sigs/kueue
+cd kueue/cmd/experimental/kueue-viz
+KUEUE_VIZ_HOME=$PWD
+kubectl create ns kueue-viz
+cd backend && make && cd ..
+cd frontend && make && cd ..
+```
+
+## Authorize
+Create a cluster role that just has read only access on
+`kueue` objects and pods, nodes and events.
+
+Create the cluster role:
+
+```
+kubectl create clusterrole kueue-backend-read-access --verb=get,list,watch \
+         --resource=workloads,clusterqueues,localqueues,resourceflavors,pods,workloadpriorityclass,events,nodes
+```
+
+and bind it to the service account `default` in the `kueue-viz` namespace:
+
+```
+kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kueue-backend-read-access-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kueue-backend-read-access
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: kueue-viz
+EOF
+```
+## Run
+
+In a first terminal run:
+```
+cd backend && make debug
```

Comment:
```suggestion
cd backend && make run
```
Can we change to run? Or we need to add guide how to run with CompileDaemon.

Reply:
regarding this one I don't know. I would prefer to keep `make debug` for now as the preferred documented target to encourage testing and contributions.
When we have images and releases ready, maybe, we can revert to `make run`

On the other hand, we can say that we should have this in the CONTRIBUTING.md guides.

If we remain with `make debug` in `README.md`, I can just add `go get github.com/githubnemo/CompileDaemon` in the pre-req ?
wdyt ?

Reply:
I have pushed an update with `CompileDaemon` installation instructions.